python/GafferSceneUI/SetUI.py

```diff
@@ -136,6 +136,23 @@ def __setValue( plug, value, *unused ) :
 	with Gaffer.UndoScope( plug.ancestor( Gaffer.ScriptNode ) ) :
 		plug.setValue( value )
 
+def __spreadsheetTargetNode( plug ) :
+
+	# Find the first node the plug's column in connected to
```

Comment:
in connected to -> is connected to

Reply:
Doh! fixed [here](https://github.com/GafferHQ/gaffer/pull/4043/commits/7e1ecc4ef54ee17652f29e1fb6fc2ddf667254b7). Thanks.